src/sveltekit/templates.ts

```diff
@@ -90,6 +90,8 @@ Feel free to delete this file and the entire sentry route.
 
 <script>
   import * as Sentry from '@sentry/sveltekit';
+  
+  let hasSentError = false;
```

Comment:
I'm not sure how important this is on an error page but we could add the syntax for Svelte Runes here if people prefer that.

```suggestion
  // Svelte Runes (requires Svelte 5)
  // let hasSentError = $state(false);
  let hasSentError = false;
```

Reply:
nice! TIL